cfgov/v1/atomic_elements/organisms.py

```diff
@@ -900,7 +904,10 @@ class ChartBlock(blocks.StructBlock):
                   '"Data from the last six months are not final."')
     y_axis_label = blocks.CharBlock(
         required=False,
-        help_text='Custom y-axis label')
+        help_text='Custom y-axis label. '
+                  'NOTE: Line-Index chart y-axis '
```

Comment:
Huh. This new note is not accurate.

![screen shot 2018-09-07 at 1 49 19 pm](https://user-images.githubusercontent.com/1183435/45234843-ecade480-b2a4-11e8-94eb-1c69fac6ab2f.png)

![screen shot 2018-09-07 at 1 49 37 pm](https://user-images.githubusercontent.com/1183435/45234853-f2a3c580-b2a4-11e8-9dc0-b954758f2539.png)

Reply:
Uhhh that's not possible. My mistake, I see the issue. You need to run `./frontend.sh` to bump chart builder. Prior sure what you're seeing is the prior behavior before the bump. Added to testing instructions.

Reply:
... It also would have helped if I'd changed his graph from "Line" type to "Line-Index" type.

🤦‍♀️